chainntnfs/txnotifier.go

```diff
@@ -614,8 +614,8 @@ func (n *TxNotifier) RegisterConf(txid *chainhash.Hash, pkScript []byte,
 	if err == nil {
 		if hint > startHeight {
 			Log.Debugf("Using height hint %d retrieved from cache "+
-				"for %v instead of %d", hint, ntfn.ConfRequest,
```

Comment:
probably can drop `chainntnfs` logs given this is a `discovery`-heavy pull

Reply:
Downgraded to trace instead. This log is actually very helpful, which gives me another possible issue to work on. For instance, when opening a channel we always mine 6 blocks. Oftentimes, the `TxNotifer` gets a block ahead of the gossip system. Say we init a channel open at height 500, and we would wait till 506 to announce it. But the `TxNotifier` gets a height from its cache and begins with a height of 501, thus never notifying the gossiper that it's confirmed with 6 blocks since now it wants 507.

Reply:
Don't think this should be downgraded since it has helped us before in the past, my comment wasn't clear my bad

Reply:
Gotya. Switched back and made into its own commit.